CHANGELOG.md

```diff
@@ -34,9 +34,10 @@
 - [#432](https://github.com/oauth2-proxy/oauth2-proxy/pull/432) Update ruby dependencies for documentation (@theobarberbany)
 - [#471](https://github.com/oauth2-proxy/oauth2-proxy/pull/471) Add logging in case of invalid redirects (@gargath)
 - [#462](https://github.com/oauth2-proxy/oauth2-proxy/pull/462) Allow HTML in banner message (@eritikass).
-- [#412](https://github.com/pusher/oauth2_proxy/pull/412) Allow multiple cookie domains to be specified (@edahlseng)
+- [#412](https://github.com/oauth2-proxy/oauth2_proxy/pull/412) Allow multiple cookie domains to be specified (@edahlseng)
 - [#413](https://github.com/oauth2-proxy/oauth2-proxy/pull/413) Add -set-basic-auth param to set the Basic Authorization header for upstreams (@morarucostel).
 - [#483](https://github.com/oauth2-proxy/oauth2-proxy/pull/483) Warn users when session cookies are split (@JoelSpeed)
+- [#414](https://github.com/oauth2-proxy/oauth2-proxy/pull/414) Always decode the cookie regardless of config flags used (@ti-mo)
```

Comment:
I think this will also need a note in the breaking changes section of the changelog, WDYT?

Reply:
IMO this is simply a bugfix, since we didn't really alter any behaviour against the expectation of the user. See my comment below: https://github.com/oauth2-proxy/oauth2-proxy/pull/414/files#r410755332.